include/grid.h

```diff
@@ -11,11 +11,11 @@ namespace detail {
 
 	using namespace allscale::api::user::data;
 
-	inline GridPoint<1> sycl_id_to_grid_point(cl::sycl::range<1> range) { return GridPoint<1>(range[0]); }
+	inline GridPoint<1> sycl_id_to_grid_point(celerity::range<1> range) { return GridPoint<1>(range[0]); }
```

Comment:
Should we take this opportunity to fix the argument type to be an actual id, instead of a range, and maybe just call it `id_to_grid_point`?

Reply:
Done both, also fixed up the call sites.